docs/admin/backup.rst

```diff
@@ -0,0 +1,177 @@
+that allows for backup and restoration of user-specified VMs. 
+
+To perform backups, you will need:
+
+ - an encrypted storage medium
+ - a secure place to store backup credentials    
+
+Backup 
+------
+
+Prepare files in ``dom0``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Files outside the ``dom0`` home folder aren't normally backed up by the 
+Qubes backup tool. Preserve key configuration files by coping them to the 
+home directory. 
+
+In a ``dom0`` Terminal via **Q ▸ Terminal Emulator**:
+
+  .. code-block:: sh
+
+    mkdir ~/backup/
+    cp -r /usr/share/securedrop-workstation-dom0-config/ ~/backup/
+
+If you have customized :doc:`clipboard access <managing_clipboard>` 
+to SecureDrop Workstation, or have made any other customizations 
+involving RPC policies, back up related configuration files:   
+
+  .. code-block:: sh
+
+   mkdir -p ~/backup/etc/qubes
+   mkdir ~/backup/etc/qubes-rpc
+   cp -a /etc/qubes/* ~/backup/etc/qubes/
+   cp -a /etc/qubes-rpc/* ~/backup/etc/qubes-rpc
+
+Back up SecureDrop Workstation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ensure your storage medium is plugged in, attached to a VM, and mounted.
+(LUKS-encrypted storage media may be attached to ``sd-usb``, or you may use a custom 
+VM of your choosing, if you are relying on another form of encryption such as 
+Veracrypt).
+
+Navigate to **Q ▸ System Tools ▸ Backup Qubes**, and move all VMs from 
+"Selected" to "Available" by pressing the ``<<`` button. 
+
+To target a VM for backup, highlight it and move it into the "Selected" 
+column by pressing the ``>`` button. Select the following VMs:
+
+ - ``dom0``
+ - ``vault``
+ - any customized VMs that you wish to preserve.
+
+You do not need to back up the ``sd-`` VMs.
+
+Click "Next", and in "Backup destination," specify the VM and directory 
+corresponding to your storage medium's current mount point.
+
+Set a strong, unique backup passphrase (7-word diceware), and ensure this 
+passphrase is stored securely outside SecureDrop Workstation. 
+
+.. note:: 
+ This passphrase protects sensitive 
+ components of your SecureDrop instance, including the *Submission Private Key*.
+ Ensure that it is a very strong password and is stored securely.   
+
+Uncheck "save backup profile," then proceed with the backup.
+
+QubesOS recommends verifying the integrity of the backup once the backup 
+completes. This can be done by using the Restore Backup GUI tool and selecting
+"Verify backup integrity, but do not restore the data." For details, see the 
+`QubesOS backup documentation <https://www.qubes-os.org/doc/backup-restore/>`_.
+
+Restore
+-------
+
+Reinstall QubesOS
+~~~~~~~~~~~~~~~~~
+
+To restore SecureDrop Workstation, follow our 
+:doc:`pre-install tasks <install>` to provision a QubesOS system complete with
+updated base templates.
+
+.. note::
+ If you backed up a number of custom VMs, during the installer, you may opt
+ to un-select "create default application qubes (personal, work, untrusted, vault)".
```

Comment:
To reduce the number of "ifs", maybe we can just say explicitly here _don't_ install those default VMs. We don't care about personal, work, or untrusted—and vault will be coming from the backup.

Reply:
oh, I remember why I had the "if" and this workflow here -- if they clean install and then need to re-download the RPM, without the default application qubes, they don't actually have a network-connected VM with which to grab the RPM, so our instructions are technically incomplete.

Reply:
(fixed in c462517)